### ISSUE TITLE
MojoMojo::Installation is a little hard to follow

### ISSUE THREAD

**Issue by f00li5h** [mojomojo/mojomojo#78]
## I installed the awesome debian package for mojomojo, 

(libmojomojo-perl - Version: 1.01+dfsg-1) and found myself wondering WTF do i do next..

```
    man mojomojo
```

just gave me the perldoc (not sure if this can be helped - maybe if it were MojoMojo::Installtion?) 

Some stuff that could be added to the docs: 
## Step 2 - configure could use:
- mention of  the env var for the config file:
  
    $  MOJOMOJO_CONFIG=./path/to/config.conf  mojomojo_spawn_db.pl
  
  since as I can't put my config in $FindBin::Bin 'cause it's installed globally and i had to rummage in the source for Config::JFDI to figure out what the name of the env var to set ...
- the cunks of Config::general in the pod could use 
  
  ```
  < Model::DBIC >  < connect_info >
  ```

around them as the hilighted code chunks look right for copy/pasting but when you do, you are just given "No valid Data Source Name (DSN)"... 
- the No DSN message could be mroe descriptive too ""No valid Data Source Name (DSN) in connect_info section of Model::DBIC in < whichever config mojomojo_spawn_db.pl read >"
## I'm not sure if it's deployed, and nor is mojomojo

```
  $  MOJOMOJO_CONFIG=./path/to/config.conf  mojomojo_spawn_db.pl
    Deploying schema to dbi:SQLite:./mojomojo.db
    Creating initial data
    Success!
```
- after deploying the database
  
    $  MOJOMOJO_CONFIG=./MojoMojo.conf   mojomojo_server.pl -p3000

tells me that

```
***********************************************
ERROR. Looks like you need to deploy a database.
Run script/mojomojo_spawn_db.pl
***********************************************
```

and yet

```
    $  MOJOMOJO_CONFIG=./MojoMojo.conf   mojomojo_spawn_db.pl
```

says that

```
    You have already deployed your database
```

so for a second opinion

```
 # apt-get install sqlite3
 $ echo '.dump' | sqlite3 ./mojomojo.db |grep CREATE |wc -l
 47
```

I think I have a database ... but it's hard to say

 this might be a bug with the debian package, or with my understanding...
## tl;dr
- i'm easily confused when i have to think about what i'm copy/pasting
- error messages are less helpful than they coudl be
- the scripts seem to have different ideas of what "having a database" means 

this was too long for IRC, and I don't know enough to make it seperate bugs ...
